docs/content/manual/pre-release/upgrade/auto-upgrade-engine.md

```diff
@@ -13,9 +13,17 @@ title: Automatically Upgrading Longhorn Engine Test
 1. Create a StatefulSet of scale 10 using 10 Longhorn volume. Set node selector so that all pods land on the same node.
 2. Upgrade Longhorn to use a newer default engine image
 3. In Longhorn UI and Longhorn manager logs, Verify that Longhorn doesn't upgrade all volumes at the same time. Only 3 at a time.
-
+4. In longhorn UI, while the concurrent value is set to limit 1, change the value to higher number and verify if the upgrade is happening as per the new value.
+5. Create stateful set pods with volume claim template, set the scale to 10 and in the configure options, select RWO and RWX. Verify the volumes created have RWO set
```

Comment:
RWO and RWX shouldn't be set at the same time? Not sure what's the logic on the Rancher side.

Reply:
I think that RWX and RWO can be set at the same time. Because [accessModes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims) is an array. In this case, Longhorn should create a RWX volume as see in [the code](https://github.com/longhorn/longhorn-manager/blob/c2e1728c2dbfb05f8060bc182c9ad5c23d4171ee/csi/controller_server.go#L129). Can you verify it @anupama2501 

But yeah, they shouldn't be set at the same time.

Reply:
Sorry, that was a mistake. Yes @PhanLe1010  RWX takes precedence.

Reply:
A PVC's access mode is a request for storage (PV) that at least supports the required modes.
Here is some more explanation on this: https://stackoverflow.com/questions/60248790/why-can-you-set-multiple-accessmodes-on-a-persistent-volume

The reason we set longhorns access mode to RWX if the claim contains RWX, is that we need to be able to guarantee that the volume can be accessed in RWX mode if one of the possible access modes is RWX.